topdown/crypto.go

```diff
@@ -16,15 +19,15 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"encoding/pem"
+	"errors"
 	"fmt"
-	"hash"
-	"os"
-	"strings"
-
 	"github.com/open-policy-agent/opa/ast"
 	"github.com/open-policy-agent/opa/internal/jwx/jwk"
 	"github.com/open-policy-agent/opa/topdown/builtins"
 	"github.com/open-policy-agent/opa/util"
+	"hash"
+	"os"
+	"strings"
```

Comment:
todo: the imports in this file appear to have been mixed together, generally the std lib ones are first.